fcrepo-kernel-impl/src/test/java/org/fcrepo/integration/kernel/impl/FedoraResourceImplIT.java

```diff
@@ -202,12 +202,13 @@ public void testObjectGraph() throws RepositoryException {
         final Graph graph = object.getTriples(subjects, PropertiesRdfContext.class).asModel().getGraph();
 
         // jcr property
-        final Node s = createGraphSubjectNode(object);
+        Node s = createGraphSubjectNode(object);
         Node p = createURI(REPOSITORY_NAMESPACE + "uuid");
         Node o = createLiteral(object.getNode().getIdentifier());
-        assertTrue(graph.contains(s, p, o));
+        assertFalse(graph.contains(s, p, o));
```

Comment:
Should this be `assertFalse(graph.contains(s, p, ANY));`? Does it matter what the object of this triple is? We just don't want to see anything with this predicate, right?